package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "node-neutralino",
+  "package-name": "node-neutralino",
+  "version": "1.1.3",
+  "description": "wrapper project for neutralinojs backend app",
```

Comment:
Please use "Neutralinojs" where you refer Neutralinojs. Thanks :tada:

Reply:
Sure Sir 😊